straxen/plugins/event_processing.py

```diff
@@ -322,38 +330,63 @@ class EventPositions(strax.Plugin):
          'Correction added to r_naive for field distortion (cm)'),
         ('theta', np.float32,
          'Interaction angular position (radians)')
-    ] + strax.time_fields
+            ] + strax.time_fields
 
     def setup(self):
-        self.map = InterpolatingMap(
-            get_resource(self.config['fdc_map'], fmt='binary'))
+
+        is_CMT = isinstance(self.config['fdc_map'], tuple)
+
+        if is_CMT:
+            
+            map_algo = list(self.config['fdc_map'])
+            name_map = list(map_algo[1])
+            name_map[0] = '_'.join([name_map[0], self.config['default_reconstruction_algorithm']])
+            map_algo[1] = tuple(name_map)
+            map_algo = tuple(map_algo)
```

Comment:
If you want to have this a little simpler:
```python
cmt, cmt_conf, is_nt = config['fdc_map']
cmt_conf = (f'{cmt_conf[0]}_{self.config["default_reconstruction_algorithm"]]}' , cmt_conf[1])
map_algo = cmt, cmt_conf, is_nt

Reply:
Thanks, much better!

Reply:
As far as I can see, you did not change it though 😉

Reply:
I must be getting senile, I was really sure I implemented it 😅